### PR TITLE
duckstation: 0.1.6658 -> 0.1.6759

### PR DESCRIPTION
**Pull request by AndersonTorres**
## Description of changes

Triggered by https://github.com/NixOS/nixpkgs/pull/322058 and  https://github.com/NixOS/nixpkgs/pull/318574

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/du/duckstation/sources.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        postFetch = ''
+          cd $out
+          mkdir -p .nixpkgs-auxfiles/
+          git rev-parse HEAD > .nixpkgs-auxfiles/git_hash
+          git rev-parse --abbrev-ref HEAD | tr -d '\r\n' > .nixpkgs-auxfiles/git_branch
+          git describe --dirty | tr -d '\r\n' > .nixpkgs-auxfiles/git_tag
+          git log -1 --date=iso8601-strict --format=%cd > .nixpkgs-auxfiles/git_date
+          find $out -name .git -print0 | xargs -0 rm -fr
+        '';
+        hash = "sha256-HETo7mChBASnr5prPUWcOhS4TIESFdrs1haEXQpnuzs=";
```
Comment (SuperSamus) [inline pkgs/by-name/du/duckstation/sources.nix:35]: I'd add:
```
# NOTE: Don't forget to change the hash in shaderc-patched.nix as well.
```

**Review by qubitnano — APPROVED** [review]
Result of `nixpkgs-review pr 322093` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>duckstation</li>
  </ul>
</details>

File: pkgs/by-name/du/duckstation/sources.nix (hunk 1 above)
Comment (AndersonTorres) [reply pkgs/by-name/du/duckstation/sources.nix:35]: Maybe it's better to merge both in the same file?

**Comment by AndersonTorres** [comment]
For info, I have tried to  build the most recent version, and the truckload of errors is astounding.

**Review by qubitnano — APPROVED** [review]

**Review by thiagokokada — APPROVED** [review]

**Pull request merged**